zulip/integrations/log2zulip/log2zulip

```diff
@@ -94,20 +94,28 @@ def process_logs():
             process_lines(new_lines, filename)
             file_data["last"] += len(new_lines)
         new_data[log_file] = file_data
-    open(data_file_path, "w").write(ujson.dumps(new_data))
+    open(data_file_path, "w").write(json.dumps(new_data))
 
 if __name__ == "__main__":
+    parser = zulip.add_default_arguments(argparse.ArgumentParser())  # type: argparse.ArgumentParser3
+    parser.add_argument("--control-path", default="/etc/log2zulip.conf")
+    args = parser.parse_args()
+    # On posix systems, we set the config directory explicitly for legacy reasons.
+    if not args.zulip_config_file and os.name == "posix":
+        args.zulip_config_file = "/etc/log2zulip.zuliprc"
```

Comment:
Why the `/etc/log2zulip.zuliprc` path?  I guess I don't see the point of this when we already have `/etc/log2zulip.conf`?  If there's a good reason, this sort of thing should go into the commit message...

Reply:
I guess I see the old code did that, but I don't see any real benefit?  Might want to just drop the second path.

Reply:
Hm, I think one benefit of the old code is that you have your `zuliprc` file separated from the `log2zulip` configuration, meaning you could just download it as usual without modifying, or reuse an existing one. So I'd actually opt for keeping it this way.

If we drop it, would the `zuliprc` info rest in `log2zulip.conf`, or the other way around?